src/pages/StateHistoryCount.tsx

```diff
@@ -34,10 +34,18 @@ const SetCountButton: React.FC = () => {
 
   return (
     <>
-      <button disabled={hasNext} onClick={() => set(getCount() - 1)} data-testid="button-decrease">
+      <button
+        disabled={hasNext}
+        onClick={() => set(getCount() - 1)}
+        data-test-id="button-decrease"
```

Comment:
Could you please fix unit tests too?
It seems that when changing to data-test-id, the UT will fail.

Reply:
@mvn-luanvu-hn 

Ah, that's a good point, thanks!
I think it's because of the name attribute of testId is not standardized and conflict between UT and playwright.(UT `data-testid`, playwright `data-test-id`)

I found the way to set customized testId attribute in playwright, so are you ok to standardize to name `data-testid` for all testing when you use testId attribute? cc: @jinmayamashita

Reply:
> so are you ok to standardize to name data-testid for all testing when you use testId attribute

I agree with `data-testid` 👍🏼 because it is one less character 😄